File: java/src/main/java/com/experian/dvs/client/server/address/search/RestApiAdditionalOption.java
package com.experian.dvs.client.server.address.search;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RestApiAdditionalOption {

    @JsonProperty("name")
    private String name;

    @JsonProperty("value")
    private String value;

    public RestApiAdditionalOption() {
    }

    public RestApiAdditionalOption(final String name, final String value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: csharp/DVSClient/DVSClient/server/address/search/RestApiAdditionalOption.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Search
{
    public class RestApiAdditionalOption
    {
        [JsonProperty("name")]
        public string? Name { get; set; }

        [JsonProperty("value")]
        public string? Value { get; set; }

        public RestApiAdditionalOption()
        {
        }

        public RestApiAdditionalOption(string? name, string? value)
        {
            Name = name;
            Value = value;
        }
    }
}